the-experts-BE/app.js

```diff
@@ -33,6 +39,47 @@ app.post("/addContractor", function(req, res, next) {
   );
 });
 
+app.post("/addReview", function(req, res, next) {
+  queries.addReview(req.body, (err, dataResponse) => {
+    if (err) next(err);
+    res.json(dataResponse);
+  });
+});
+
+app.post("/login", function(req, res, next) {
+  queries.getUser(req.body.email, (err, dataResponse) => {
+    if (err) next(err);
+    comparePasswords(req.body.password,dataResponse.password,(error,result)=>{
+      if(error) next(error);
+      if(!result) res.json(null);
+      const jwt=sign(req.body.email,SECRET);
+      res.cookie('jwt',jwt);
+    res.json(jwt);
+  });
+});
+});
+
+app.post("/signup", function(req, res, next) {
+  let body = {
```

Comment:
No `let`s if the variable isn't being mutated!!

Also you could shorten this with ES6 spread syntax:

```js
const body = { ...req.body }
```